bin/mock_onfido.pl

```diff
@@ -32,14 +32,14 @@
 ################################################################################
 # applicants
 # create applicant
-post '/v2/applicants' => sub {
+post '/v3.4/applicants' => sub {
```

Comment:
is 3.4 an alpha version ?

Reply:
the migration guide mentions v3.4 https://developers.onfido.com/guide/api-v2-to-v3-migration-guide